src/models/LoginRequest.ts

```diff
@@ -0,0 +1,4 @@
+export type LoginRequest = {
+    email: String,
+    password: String
```

Comment:
trailing commas all through this pr please https://www.typescriptlang.org/docs/handbook/enums.html